qe-on-duty-reporter/qe_on_duty/gh_client.py

```
@@ -0,0 +1,207 @@
+
+        Args:
+            gh_path: Path to gh CLI binary (default: "gh" in PATH)
+            max_retries: Maximum number of retries for failed commands
+        """
+        self.gh_path = gh_path
+        self.max_retries = max_retries
+        self._verify_gh_auth()
+
+    def _verify_gh_auth(self):
+        """Verify gh CLI is authenticated."""
+        result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)